gradle.properties

```diff
@@ -1,3 +1,3 @@
 group=com.wepay.waltz
 sourceCompatibility=1.8
-version=0.5.0
+version=0.5.1
```

Comment:
This is out of date now. We can probably leave this unchanged

Reply:
Oops forgot to remove this change. Will fix it.

Reply:
fixed.